src/views/Shopping/Shopping.jsx

```diff
@@ -0,0 +1,77 @@
+import { useReducer } from 'react';
+import AddItem from '../../components/AddItem/AddItem';
+import ItemList from '../../components/ItemList/ItemList';
+
+const nextId = 6;
```

Comment:
because this is hard coded number instead of a function all the user's new items they have have the same id. so if the user edits one of their new items, they edit all of them.